service/src/main/java/com/epam/esm/service/impl/TagServiceImpl.java

```diff
@@ -1,47 +1,55 @@
 package com.epam.esm.service.impl;
 
-import com.epam.esm.dao.entity.Tag;
-import com.epam.esm.dao.dao.impl.TagDaoImpl;
+import com.epam.esm.dao.dao.TagDao;
+import com.epam.esm.model.entity.Tag;
 import com.epam.esm.service.CommonService;
 import com.epam.esm.service.TagService;
+import com.epam.esm.service.exeption.RecourseExistException;
+import com.epam.esm.service.exeption.RecourseNotExistException;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
 import java.util.Optional;
 
 @Service
 public class TagServiceImpl implements CommonService<Tag>, TagService {
```

Comment:
'TagServiceImpl' should implement only 'TagService` -  'TagService' is based on CommonService

Reply:
@valevaty98 done

Reply:
@valevaty98 done